inc/crontask.class.php

```diff
@@ -1780,6 +1786,16 @@ static function cronTemp(CronTask $task) {
          }
       }
 
+      //second step drop director if empty
+      foreach ($files as $filename) {
+         if (is_dir($filename) && is_readable($filename)
+            && count(scandir($filename)) == 2) {
+            if (@rmdir($filename)) {
+               $nb++;
+            }
+         }
+      }
+
```

Comment:
As you use `RecursiveIteratorIterator::CHILD_FIRST` flag, you should be able to use `rmdir` directly in first loop.

Reply:
done